sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -58,7 +58,7 @@ class VersionsSuite extends SparkFunSuite with Logging {
    */
   protected def withTempDir(f: File => Unit): Unit = {
     val dir = Utils.createTempDir().getCanonicalFile
-    try f(dir) finally Utils.deleteRecursively(dir)
+    f(dir)
```

Comment:
Hm, actually, shall we just skip the failing tests for now (`assume(Utils.isWindows)`)? Given what we talked, only few tests are failed in some few conditions when Hive version is 0.12?

Reply:
Actually, only one(2nd test mentioned above). Probably, `assume(!(Utils.isWindows && version == "0.12"))` would be ok. WDYT?

Reply:
Yea, sounds good.